enterprise_access/apps/subsidy_access_policy/content_metadata_api.py

```diff
@@ -111,21 +111,25 @@ def get_list_price_for_content(enterprise_customer_uuid, content_key, content_me
             raise ContentPriceNullException(f'Could not determine list price for {content_key}') from exc
 
     # Note that the "content_price" key is guaranteed to exist, but the value may be None.
-    return list_price_dict_from_usd_cents(content_metadata['content_price'])
+    return make_list_price_dict(integer_cents=content_metadata['content_price'])
 
 
-def list_price_dict_from_usd_cents(list_price_integer_cents):
+def make_list_price_dict(decimal_dollars=None, integer_cents=None):
     """
     Helper to compute a list price dictionary given the non-negative price of the content in USD cents.
```

Comment:
nit: docstring could be updated, if this method now deals with either `decimal_dollars` or `integer_cents` (and not just "USD cents").